src/components/portfolio/Portfolio.js

```diff
@@ -22,7 +25,108 @@ class Portfolio extends Component {
     )
   };
 
+  constructor(props) {
+    super(props);
+    this.state = {
+      stockDetails: [],
+      cryptoDetails: [],
+      isLoading: true,
+      showStart: false
+    };
+    this.onDelete = this.onDelete.bind(this);
+  }
+
+  onDelete(index) {
+    let stateObj = this.state;
+    let stockDetails = stateObj.stockDetails;
+    stockDetails.splice(index, 1);
+    stateObj.stockDetails = stockDetails;
+    this.setState(stateObj);
+    let portfolioDetails = {};
+    portfolioDetails.stockDetails = stockDetails;
+    AsyncStorage.setItem("portfolioDetails", JSON.stringify(portfolioDetails));
```

Comment:
We should create an abstraction for AsyncStorage, and use that service in the component that way later it will be easy to migrate the storage mechanism